packages/react-ux-capture-example/src/Lazy.jsx

```diff
@@ -17,13 +16,8 @@ export default class Lazy extends React.Component {
 	}
 	render() {
 		if (!this.state.loaded) {
-			return null;
+			return <React.Fragment>{this.props.fallback}</React.Fragment> || null;
 		}
-		return (
-			<div>
-				This was loaded lazily in {this.props.delay}ms
-				<Inline mark={this.props.mark} />
-			</div>
-		);
+		return <React.Fragment>{this.props.children}</React.Fragment>;
```

Comment:
this `<React.Fragment>` is probably not needed -  `return this.props.children` is valid

Reply:
Trying to remember why I put all of the `<React.Fragment>`s here. Nothing comes to mind ;)